### PR TITLE
Migrate to f string.

### PR DESCRIPTION
**Pull request by ShreehariVaasishta**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

## Description of the pull request
As per issue #45, migrating to f string.

### CODE REVIEW

File: stream_chat/async_chat/channel.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ async def truncate(self):
 
         :return: The server response
         """
-        return await self.client.post("{}/truncate".format(self.url))
+        return await self.client.post("{self.url}/truncate")
```
Comment (ferhatelmas) [inline stream_chat/async_chat/channel.py:140]: f

File: stream_chat/async_chat/channel.py [hunk 2]
```diff
@@ -193,7 +193,7 @@ async def mark_read(self, user_id, **data):
         :return: The server response
         """
         payload = add_user_id(data, user_id)
-        return await self.client.post("{}/read".format(self.url), data=payload)
+        return await self.client.post("{self.url}/read", data=payload)
```
Comment (ferhatelmas) [inline stream_chat/async_chat/channel.py:196]: f

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
Thanks for PR.

Could you see the comments and fix them too ?

**Comment by ShreehariVaasishta** [comment]
Sure

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Pull request merged**